config/vufind/config.ini

```diff
@@ -2159,6 +2159,9 @@ HMACkey = mySuperSecretValue
 route[record] = collection
 route[search2record] = search2collection
 
+; On collection list tab, display all collection items also on lower archive levels
+; showFullHierarchyTree = true
```

Comment:
Would it be better to move this into the hierarchy configuration in HierarchyDefault.ini, and then access the setting through the hierarchy driver retrieved from the record driver? This feels like a setting that might better belong in that context, and moving it would also eliminate the need to change factories and constructors, since you could just add a new method to [the ConfigurationBased hierarchy driver](https://github.com/vufind-org/vufind/blob/dev/module/VuFind/src/VuFind/Hierarchy/Driver/ConfigurationBased.php) and then access it using `$driver->getHierarchyDriver()->...`.